### PR TITLE
pillow retry fix off 2016 09 26 23.06 production deploy

### PR DESCRIPTION
**Pull request by snopoke**
@esoergel

### CODE REVIEW

File: corehq/ex-submodules/pillow_retry/tests.py [hunk 1]
```diff
@@ -27,6 +28,8 @@ def FakePillow():
 
 
 def create_error(change, message='message', attempts=0, pillow=None, ex_class=None):
+    change = force_to_change(change)
+    change.metadata = ChangeMeta(data_source_type='couch', data_source_name='test_commcarehq', document_id=change.id)
```
Comment (dimagimon) [inline corehq/ex-submodules/pillow_retry/tests.py]: E501 line too long (117 > 115 characters)

File: corehq/ex-submodules/pillow_retry/tests.py [hunk 2]
```diff
@@ -73,49 +76,25 @@ def test_attempts(self):
...
-        get = PillowError.get_or_create({'id': id}, FakePillow(), meta)
-        self.assertEqual(None, get.doc_date)
-
     def test_get_errors_to_process(self):
         # Only re-process errors with
         # current_attempt < setting.PILLOW_RETRY_QUEUE_MAX_PROCESSING_ATTEMPTS
         date = datetime.utcnow()
         for i in range(0, 5):
-            error = create_error({'id': i}, attempts=i+1)
+            error = create_error({'id': i, 'doc': {}}, attempts=i+1)
```
Comment (dimagimon) [inline corehq/ex-submodules/pillow_retry/tests.py]: E226 missing whitespace around arithmetic operator

File: corehq/ex-submodules/pillow_retry/tests.py [hunk 3]
```diff
@@ -221,7 +200,7 @@ def test_deleted_doc(self):
 
     def test_bulk_reset(self):
         for i in range(0, 5):
-            error = create_error({'id': i}, attempts=settings.PILLOW_RETRY_QUEUE_MAX_PROCESSING_ATTEMPTS)
+            error = create_error({'id': i, 'doc': {}}, attempts=settings.PILLOW_RETRY_QUEUE_MAX_PROCESSING_ATTEMPTS)
```
Comment (dimagimon) [inline corehq/ex-submodules/pillow_retry/tests.py]: E501 line too long (116 > 115 characters)

**Comment by snopoke** [comment]
:white_check_mark:

**Pull request merged**